### PR TITLE
Added reference material

### PR DESCRIPTION
**Pull request by abkro**
Added references on AppArmor, backporting kernel drivers and understanding init scripts. When the section on Halium 9 porting starts to take shape, the reference material should probably be moved up one level, but for now, it should be OK to put it here.

### CODE REVIEW

File: porting/halium_7-1/Reference_material.rst [hunk 1]
```diff
@@ -40,4 +40,27 @@ Form factor
...
+Depending on your device port, what kernel kernel and Halium versions it is based on, you may need to backport one or more kernel drivers. The procedure for backporting bluetooth to Halium 7.1-based ports is described in `the porting-FAQ <https://pad.ubports.com/p/porting-faq>`_ . The process is challenging and has potential pitfalls. 
+
+If you wish to gain a better understanding of backporting, please refer to `the Backports Project wiki page <https://backports.wiki.kernel.org/index.php/Main_Page>`_
+
+.. _Understanding-init-scripts:
+
+Understanding init scripts
+--------------------------
+
+The init scripts play a key role in bringing up the different services that need to run on your device in order for it to function properly. Refer to `the Android init language <https://android.googlesource.com/platform/system/core/+/master/init/README.md>`_ for an explanation of the syntax of these scripts.
```
Comment (doniks) [inline porting/halium_7-1/Reference_material.rst:66]: how about "... different services inside your android container that need to run ..."?

**Review by doniks — CHANGES_REQUESTED** [review]

File: porting/halium_7-1/Reference_material.rst [hunk 2]
```diff
@@ -40,4 +40,27 @@ Form factor
...
+--------
+
+To understand Apparmor, what it is and how it works, refer to `AppArmor on the Ubuntu Wiki <https://wiki.ubuntu.com/AppArmor>`_
+
+.. _Backporting-kernel-drivers:
+
+Backporting kernel drivers
+--------------------------
+
+Depending on your device port, what kernel kernel and Halium versions it is based on, you may need to backport one or more kernel drivers. The procedure for backporting bluetooth to Halium 7.1-based ports is described in `the porting-FAQ <https://pad.ubports.com/p/porting-faq>`_ . The process is challenging and has potential pitfalls.
```
[the remark this answers is not in the document]
Comment (abkro) [reply porting/halium_7-1/Reference_material.rst]: Corrected.

File: porting/halium_7-1/Reference_material.rst [hunk 3]
```diff
@@ -40,4 +40,27 @@ Form factor
...
+--------------------------
+
+Depending on your device port, what kernel kernel and Halium versions it is based on, you may need to backport one or more kernel drivers. The procedure for backporting bluetooth to Halium 7.1-based ports is described in `the porting-FAQ <https://pad.ubports.com/p/porting-faq>`_ . The process is challenging and has potential pitfalls. 
+
+If you wish to gain a better understanding of backporting, please refer to `the Backports Project wiki page <https://backports.wiki.kernel.org/index.php/Main_Page>`_
+
+.. _Understanding-init-scripts:
+
+Understanding init scripts
+--------------------------
```
[the remark this answers is not in the document]
Comment (abkro) [reply porting/halium_7-1/Reference_material.rst]: Changed title

File: porting/halium_7-1/Reference_material.rst (hunk 1 above)
Comment (abkro) [reply porting/halium_7-1/Reference_material.rst]: Included this.

**Review by doniks — APPROVED** [review]
thx

**Pull request merged**